runner.py

```diff
@@ -0,0 +1,174 @@
+linus_torvalds = Character("Linus Torvalds", "Don't break userspace around him", [unix_based_operating_system, glock], 100, 10, "Greetings, I’m Linus Torvalds, the guy who thinks a Unix-based system can solve nearly anything, as long as you’ve got the right mindset. And yes, I’ve got a Glock for when things get intense. Don’t worry, it’s metaphorical—just a reminder that even in the tech world, you’ve got to be prepared for anything. Ready to dive into the power of Unix?", "human", 40)
+merchant_of_death = Character("Merchant", "Got traded for Brittney Griner", [sword, grenade], 100, 10, "I am the Merchant of Death, and let’s just say I have a flair for the dramatic. Whether it's a sword or a grenade, I’ve got the tools to make a statement. Life’s short, so why not live it on the edge? If you’re looking for excitement, danger, and maybe a touch of chaos, you’ve come to the right place. Now, who’s up for a bit of mayhem?", "death", 20)
+
+# Scenes
+suburban_neighborhood = Scene("Suburban Neighborhood", "A quiet suburban neighborhood with neatly trimmed lawns and white picket fences.", [terry_davis], [shasta], sub_graphic)
+forest = Scene("Forest", "A dense forest with towering trees and the sound of birds chirping.", [chris_chan], [stick, water_bottle], forest_graphic)
+locked_gate = Scene("Locked Gate", "A large, imposing gate that stands between you and your goal.", [linus_torvalds], [coffee], gate_graphic)
+merchant_scene = Scene("Merchant's Hideout", "A hidden hideout where the Merchant of Death resides.", [merchant_of_death], [blood_of_christ], merchant_graphic)
+
+# Player Setup
+while(True):
+    print("_______   ________  ________  ________")
+    print("_______   ________  ________  ________")
+    print("_______   ________  ________  ________")
+    player_name = input("\nWhat is your name, warrior? ")
+    if len(player_name) > 26:
+        print("Your name is too long! Please choose a shorter name.")
+    elif len(player_name) <= 1:
+        print("Your name is too short! Please choose a longer name.")
+    else:
+        print(knight_graphic)
+        print(f"Good luck on your journey, {player_name.title()}!")
+        #for i in range(3):
+        #    sleep(1)
+        #    print(".")
+        sleep(1)
+        break
+
+player = Player(player_name, 100, 10, [])
+
+# Game Loop
+current_scene = suburban_neighborhood
+
+def move_to_scene(scene):
+    global current_scene
+    current_scene = scene
+    current_scene.describe()
+
+def check_for_win(player):
+    if all(item in [i.name.lower() for i in player.inv] for item in ['commodore 64', 'sonic memorabilia', 'unix-based operating system']):
+        print("You have collected all the necessary items!")
+        print(binary_tree_graphic, end="")
+        print("The gate opens, revealing the legendary Binary Tree of Life. You have won the game!")
+        exit()
+
+def game_loop():
```

Comment:
Nit - there isn't really a puzzle in your game, but I like the concept of having to collect all the different items to win. We'll count it!